docs/_sections/artifacts.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: Artifacts
```

Comment:
Should this also be "Common DV Artifacts"?

Reply:
wanted to keep less wordy for now - particularly for purpose of sidebar